libs/features/components/banners/src/lib/banners/elewa-group-image-and-text-banner/elewa-group-image-and-text-banner.component.ts

```diff
@@ -0,0 +1,14 @@
+import { Component,Input } from '@angular/core';
+
+@Component({
+  selector: 'elewa-group-elewa-group-image-and-text-banner',
+  templateUrl: './elewa-group-image-and-text-banner.component.html',
+  styleUrls: ['./elewa-group-image-and-text-banner.component.scss'],
+})
+export class ElewaGroupImageAndTextBannerComponent {
+  @Input() title:string | undefined
+  @Input() paragraph:string| undefined
+  @Input() imgSrc:string | undefined 
+  @Input() position:string | undefined
```

Comment:
@Input() position: 'left' | 'right' = 'right';

default position is right. Check issue again.